DATA/production/qc-async/its.json

```diff
@@ -39,7 +39,8 @@
           "layer": "1111111",
           "nThreads": "1",
 	  "nBCbins" : "103",
-	  "dicttimestamp" : "0"
+	  "dicttimestamp" : "0",
+	  "geomPath" : "/home/epn/odc/files/"
```

Comment:
Ciao @iravasen , this cannot work, we don't have the epn files. 
If this is o2sim_geometry-aligned.root, then it is local to the job, you can take it from there. See e.g. https://github.com/AliceO2Group/O2DPG/blob/ddc539d16a12e62b615c79f1a917047ba7bc02a0/DATA/production/qc-async/itstpctof.json#L47